### PR TITLE
Refactor/stickers controller QA

### PR DESCRIPTION
**Pull request by PalauiDCL**
## QA 
This PR only exists in order to create a testing ground for QA
The stickers refactor changed how stickers are instantiated, but there should be no visible changes for them

Original PR: https://github.com/decentraland/unity-renderer/pull/3298

### CODE REVIEW

**Review by kwspgoliasz — CHANGES_REQUESTED** [review]
No visual issues were found, however, the PR introduced a new issue with particles for emotes spawning much lower from their intended position - https://user-images.githubusercontent.com/110464313/198255200-d0bbd56e-11e2-496c-b88f-77a31cebe977.mp4.

The issue occurs for the following emotes: Clap, Kiss, Snowfall, Money, Ho ho ho!.

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                Destroy(this);
+                return;
+            }
+
+            poolable = poolableObject;
+            index = Random.Range(0, INDEX_AMOUNT);
+            initialized = false;
+        }
+
+        public void FixedUpdate()
```
Comment (lorux0) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs:30]: why FixedUpdate? what about Update?

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+using UnityEngine;
+
+namespace DCL
+{
+    public class ReleaseParticlesOnFinish : MonoBehaviour
+    {
+        private const int INDEX_AMOUNT = 10;
```
Comment (lorux0) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs:7]: i would rename to: `FRAME_RATE_CHECK`

**Review by kwspgoliasz — CHANGES_REQUESTED** [review]
Fix failed, as the emote particles are still spawned below the ground level. No additional issues were found.

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs (hunk 1 above)
Comment (PalauiDCL) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs:30]: @lorux0  It was like this before and I found no reason to change it ,as goes through the physics cycle, should be no issue having it here, if you feel a change is due, would be fine for me

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs (hunk 2 above)
Comment (PalauiDCL) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs:7]: @lorux0 I am not sure about the proposed change, as is not related to any rate and I am using a system similar to the AI update behaviors for checking only on the index for the calculation.
I can try to find a better name though!

**Review by kwspgoliasz — CHANGES_REQUESTED** [review]
The issue with the emote particles spawning below the ground no longer occurs.
In the latest branch, a new issue was introduced causing the emote particles to spawn too high. The issue occurs for all of the emotes with particles. The videos from both the branch (https://user-images.githubusercontent.com/110464313/200006235-8f45d26e-b86c-4543-bfa0-4aeb69a6d3e9.mp4) and the production configurations (https://user-images.githubusercontent.com/110464313/200006297-90d9662e-3627-425e-ac9c-16130c61e678.mp4) were added for comparison.

**Comment by PalauiDCL** [comment]
> The issue with the emote particles spawning below the ground no longer occurs. In the latest branch, a new issue was introduced causing the emote particles to spawn too high. The issue occurs for all of the emotes with particles. The videos from both the branch (https://user-images.githubusercontent.com/110464313/200006235-8f45d26e-b86c-4543-bfa0-4aeb69a6d3e9.mp4) and the production configurations (https://user-images.githubusercontent.com/110464313/200006297-90d9662e-3627-425e-ac9c-16130c61e678.mp4) were added for comparison.

@kwspgoliasz The issue should be corrected now ^^

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs (hunk 1 above)
Comment (lorux0) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs:30]: i think there is not reason to keep in a fixed step updating, it should work through the frame rate loop

Comment (PalauiDCL) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs:30]: Will update it then ^^

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs (hunk 2 above)
Comment (lorux0) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs:7]: ok!

**Review by kwspgoliasz — CHANGES_REQUESTED** [review]
The issues were mostly fixed. The particles for the Clap, Kiss and Money emotes are displayed correctly.
For the Snowfall emote, the snowflakes are spawned a little higher in comparison to its counterpart in the production configuration, albeit this version does not look bad.
For the Ho ho ho! emote, the particles are still spawned too low (around user's stomach instead of the mouth).
#3325_Hohoho

**Comment by xavi-dcl** [comment]
@kwspgoliasz this is ready for QA, let's close it this sprint if validation OK. Thanks.

**Comment by aixaCode** [comment]
@PalauiDCL should this PR be merked as `do not merge`. All the changes are the same as here https://github.com/decentraland/unity-renderer/pull/3298

**Review by kwspgoliasz — APPROVED** [review]
The feature appears to be implemented correctly. The particles associated with emotes, running and avatar changes seem to be displayed correctly. All of the issues regarding the particle spawn were fixed.

**Pull request merged**